components/core/CMakeLists.txt

```diff
@@ -1,6 +1,19 @@
 cmake_minimum_required(VERSION 3.16.3)
+
+set(CMAKE_TOOLCHAINS_DIR
```

Comment:
* Do we really need this variable?
* Don't forget we need to prefix it with the project name.